scratch-arduino-extensions/scratch-vm/src/extensions/scratch3_arduino/index.js

```diff
@@ -88,6 +85,8 @@ class IncludeRobot {
      * @type {Runtime}
      */
     this._runtime = runtime;
+
+    this._device = new Peripheral(this._runtime, IncludeRobot.EXTENSION_ID);
```

Comment:
I will keep the same device singleton at least, in this way we can still reuse the same connection for both extension

Reply:
I hoped but unfortunately we can't.
The  `Peripheral` needs the ID of the extension to "register" the popup to the button.

Reply:
What is the objective to maintain a single BLE connection? 
If it is related to be able to manage both the extensions, the same result could be obtained even with two different connections. This is possible by connecting both the extensions.

This is an example:

![image](https://github.com/arduino/include-robot/assets/9201530/6bc11aa4-ccb5-4513-b7af-0a36f64b03bc)